tests/functional/pytest_docker_fixtures.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+# pytest-docker fixtures
+# https://github.com/avast/pytest-docker#available-fixtures
```

Comment:
Should we make it a module docstring? I think then people can see it in IDEs when hovering over imports.
```suggestion
"""
pytest-docker fixture overrides.
See https://github.com/avast/pytest-docker#available-fixtures.
"""
import pytest
```

Reply:
Good idea.